README.md

```diff
@@ -4,7 +4,20 @@ A plugin for IntelliJ IDEA that detects outdated comments in Java code
 
 ## Launch instructions
 
-- Inside the `src/main/kotlin/...models/config/ModelFilesConfig.kt` file 
+### Plugin
+
+- Inside the `comment-updater-plugin/src/main/kotlin/...models/config/ModelFilesConfig.kt` file 
 replace `DATA_DIR` value with your local path to the directory, containing files from [here](https://drive.google.com/drive/folders/1E4XOJHfEWOlHXBPxPu3_bkKfuU4G_N49?usp=sharing)
   
-- Run  `gradle/intellij/runIde` task
\ No newline at end of file
+- Run  `gradle/comment-updater-plugin/intellij/runIde` task
+
+### Headless Plugin 
+
+- Inside the `comment-updater-plugin/src/main/kotlin/...models/config/ModelFilesConfig.kt` file
```

Comment:
It would be nice to allow users to configure these paths in the console. Correct me if I'm wrong, but the headless part of the plugin could be run in the console. So, I suggest adding some parameters responsible for configuring the paths.

Reply:
Done! Headless mode now runs with comment updater.sh and you should provide paths as common arguments. Although, you still required to changed `ModelFilesConfig.kt` if you want to run plugin (not headless part).